config/routes.php

```diff
@@ -1,50 +1,52 @@
 <?php
-use Cake\Routing\RouteBuilder;
+
 use Cake\Routing\Route\DashedRoute;
-use Cake\Routing\Router;
+use Cake\Routing\RouteBuilder;
 
-Router::plugin('DebugKit', ['path' => '/debug-kit'], function (RouteBuilder $routes) {
-    $routes->setExtensions('json');
-    $routes->setRouteClass(DashedRoute::class);
+return static function (RouteBuilder $outer_routes) {
```

Comment:
Conventionally variable names should be `camelBacked` case.

Reply:
indeed, totally forgot about that